protocol/rpcconsumer/rpcconsumer.go

```diff
@@ -583,10 +583,13 @@ rpcconsumer consumer_examples/full_consumer_example.yml --cache-be "127.0.0.1:77
 				RelaysHealthIntervalFlag:    viper.GetDuration(common.RelayHealthIntervalFlag),
 				DebugRelays:                 viper.GetBool(DebugRelaysFlagName),
 				DisableConflictTransactions: viper.GetBool(common.DisableConflictTransactionsFlag),
-				DisableRetryOnNodeErrors:    viper.GetBool(common.DisableRetryOnNodeErrorsFlag),
 				OfflineSpecPath:             viper.GetString(common.UseOfflineSpecFlag),
 			}
 
+			// set relay processor's global params
+			relayCountOnNodeError = viper.GetInt(common.SetRelayCountOnNodeErrorFlag)
+			disableRelayRetry = viper.GetBool(common.DisableRetryOnNodeErrorsFlag)
```

Comment:
use cmdRPCConsumer.Flags().IntVar(&relayCountOnNodeError) instead of cmdRPCConsumer.Flags().Int(common.SetRelayCountOnNodeErrorFlag, 2, "set the number of retries attempt on node errors") and you don't need relayCountOnNodeError = viper.GetInt(common.SetRelayCountOnNodeErrorFlag)

Reply:
nice didn't know this feature.